src/main/java/appeng/core/localization/Tooltips.java

```diff
@@ -1,19 +1,27 @@
 package appeng.core.localization;
 
+import java.text.DecimalFormat;
+import java.text.DecimalFormatSymbols;
+import java.text.MessageFormat;
 import java.util.Arrays;
 import java.util.List;
 
 import org.jetbrains.annotations.NotNull;
 
 import net.minecraft.ChatFormatting;
-import net.minecraft.network.chat.Component;
-import net.minecraft.network.chat.TextComponent;
+import net.minecraft.network.chat.*;
+
+import appeng.api.config.PowerUnits;
 
 /**
  * Static utilities for constructing tooltips in various places.
  */
 public final class Tooltips {
 
+    private static final DecimalFormat format = (DecimalFormat) DecimalFormat.getInstance();
+    private static final DecimalFormatSymbols symbols = format.getDecimalFormatSymbols();
+    private static final char SEP = symbols.getDecimalSeparator();
```

Comment:
DecimalFormat et al are not thread-safe and shouldn't be stored in a static field unless synchronized